### DISCUSSION TITLE
Admin Side Feature to Remove Test Data Users

### DISCUSSION THREAD

**Discussion by SamWadrose** [Glench/ExtPay#179] [Ideas]
I am using a test email account, not the one linked to ext pay, to test my implementation of the free trial period. I can see, on the Free Trial Users page, there is an option to show only test data. It would be nice if I could delete my testing account so I can retry the signing up for a free trial unlimited times.

Is there any other recommended way to do this?

**Comment by Glench** [comment]
Just email me! :)
